vcpkg/get_started/get-started-msbuild.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Install and use packages with MSBuild in Visual Studio
+description: Tutorial guides the user through the process of installing and using packages with MSBuild and Visual Studio.
+author: data-queue
+ms.author: danshaw2
+ms.date: 10/03/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+---
+
+# Tutorial: Install and use packages with MSBuild in Visual Studio
+
+This guide sets up a simple C++ project using the library `zlib` with the MSBuild project build system.
+
+## Prerequisites:
+- Visual Studio 2015 Update 3 or later with the C++ Desktop Development workload
+- Windows 7 or newer
+- Git
+
+## Install `vcpkg`
```

Comment:
```suggestion
## 1 - Set up vcpkg
```

Similarily, the other ## headers should be number to stay consistent with other tutorials.